pkgs/top-level/python-packages.nix

```
@@ -1429,6 +1437,12 @@ in {
 
   clustershell = callPackage ../development/python-modules/clustershell { };
 
+  clvm = callPackage ../development/python-modules/clvm { };
+
+  clvm_rs = callPackage ../development/python-modules/clvm_rs { };
+
+  clvm_tools = callPackage ../development/python-modules/clvm_tools { };
```

Please normalize the attribute name. According to the [python contributing guidelines](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines), package names within nix should follow [PEP 0503](https://www.python.org/dev/peps/pep-0503/#normalized-names)
```suggestion
  clvm-rs = callPackage ../development/python-modules/clvm-rs { };

  clvm-tools = callPackage ../development/python-modules/clvm-tools { };
```